apps/slack/frontend/src/useConnect.ts

```diff
@@ -34,12 +34,19 @@ export const useConnect = () => {
 
           await apiClient.createAuthToken(sdk, cma, token, installationUuid);
         } catch (e) {
-          console.error(e);
-          sdk.notifier.error(genericErrorMessage);
+          const error = e as Error;
+
+          // this 'AppInstallation does not exist error' is a race condition that occurs when instantiating the eventsService within the lambda.
+          // it is not a valid error to show to the user.
+          const errorMessage = JSON.parse(error?.message);
+          if (errorMessage && errorMessage.details !== 'AppInstallation does not exist.') {
+            console.error(e);
+            sdk.notifier.error(genericErrorMessage);
+          }
```

Comment:
this is where, as per the PR description, we are allowing any other error to be handled appropriately but selecting out the one ignorable error.

Reply:
I'm assuming you checked but don't want to review based off assumptions. Is there an error code or something more unique we can attach to the check rather than a string check or is this our best bet?